eng/ci/official-build.yml

```diff
@@ -45,6 +46,12 @@ extends:
     stages:     
       - stage: BuildAndTest
         jobs:
-        - template: /eng/ci/templates/official/jobs/build-test.yml@self
-        - template: /eng/ci/templates/official/jobs/linux-package.yml@self
-
+          - template: /eng/ci/templates/official/jobs/build-test.yml@self
+            parameters:
+              artifactTargetFramework: inproc6
+              jobTitle: BuildInproc6
+          - template: /eng/ci/templates/official/jobs/build-test.yml@self
+            parameters:
+              artifactTargetFramework: inproc8
+              jobTitle: BuildInproc8
+          - template: /eng/ci/templates/official/jobs/linux-package.yml@self
```

Comment:
how come we build inproc8 only for linux?

Reply:
It's not for Linux, Linux one(line 57) is a separate job.

Reply:
ohh ok that makes a lot more sense